### ISSUE TITLE
Problem with remote_hosts.cfg on Linux

### ISSUE THREAD

**Issue by sirzooro** [BOINC/boinc#3246]
People at PrimeGrid found that there is something wrong with handling of non-empty remote_hosts.cfg file on non-Windows hosts. When config is re-read, Boinc logs message like "GUI RPC request from non-allowed address 2.0.x.x". Additionally when someone connects from non-whitelisted IP, message contains invalid IP like 2.0.x.x.. On Windows IP is correct.

Original thread on PG: [https://www.primegrid.com/forum_thread.php?id=8733](https://www.primegrid.com/forum_thread.php?id=8733)

**Comment by JStateson** [comment]
This is an ongoing problem.  A discussion about this back in Feb 2018
[https://boinc.berkeley.edu/forum_thread.php?id=12268#84537](url)

It has been noted at other web sites.  It has been a while since I downloaded the boinc app from the actual Berkeley repository.   That app is really out of date.  I don't recall ever seeing the problem when the Berkeley repository was used.  The security of the install package (from whatever ppa) is up to the builder.  There is a lot of crypto mining: boinc "gridcoins" and conceivably someone may want to probe an installation.  I hope there is some other explanation.  I did find it strange that the same French IP address was used to probe my system on several of my Ubuntu setup.
